README.md

```diff
@@ -1 +1,45 @@
-# decodeMorse
\ No newline at end of file
+# decodeMorse
+
+This app decodes morse code messages:
+### morse structure
+- characters separated by a single space
+- words separated by a three spaces
+
+## Getting Started
+
+This repository includes file decode_morse.ru as main program file:
+
+## Built With
+
+- Ruby
+
+## Authors
+
+👤 **BANLON JONES**
+- Twitter: [@banlonj](https://twitter.com/banlonjones)
+- Github: [banlon-jones](https://github.com/banlon-jones)
+- LinkedIn: [banlon jones](https://www.linkedin.com/in/banlon-jones-b0205812a)
+
+👤 **HAMMAS BIN SALEEM**
+- GitHub: [@HAMMAS-SALEEM](https://github.com/HAMMAS-SALEEM)
+- Twitter: [@HammasSaleem4](https://twitter.com/HammasSaleem4)
+- LinkedIn: [HAMMAS SALEEM](https://www.linkedin.com/in/hammas-saleem)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Inspiration from Microverse
+- https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

Reply:
Thanks for the feedback
i have implemented the changes mention above